frontend/src/component/project/ProjectList/ProjectGroup.tsx

```diff
@@ -108,9 +107,7 @@ export const ProjectGroup: React.FC<{
                                             to={`/projects/${project.id}`}
                                         >
                                             <ProjectCard
-                                                onHover={() =>
-                                                    handleHover(project.id)
-                                                }
+                                                onHover={() => {}}
```

Comment:
Why empty? make it optional or remove?

Reply:
underlying component makes it required and I'd like to remove it in a separate PR